### ISSUE TITLE
Multiple buttons for a single contribution page

### ISSUE THREAD

**Issue by Vgratioulet** [adixon/ca.civicrm.contributextra#4]
When viewing a contact, on the contribution tab I have one button for my contribution page,
then I select the membership tab then the contribution tab, I have 4 buttons for the same contribution page. Then in the events tab I have 8 buttons, and if I keep on changing tabs I get I lot of buttons.

CiviCRM 4.7.18 Drupal 7.54
![contribextra_multiple_buttons](https://cloud.githubusercontent.com/assets/11544604/26360721/2a451b3c-3fd9-11e7-9c39-2bf837f6ee39.png)

**Comment by adixon** [comment]
Yeah, I think I'll have to review how 4.7 does this, there's been a change to the ajax strategy with these tabs. Thanks for the report.

**Comment by skyjacobs** [comment]
Yes, we are also getting many extra buttons. Every time a tab is opened another set is added. It would also probably make sense to move these buttons after others (record contribution, credit card, etc.), instead of before. 

![screen shot 2018-09-20 at 11 14 26](https://user-images.githubusercontent.com/43451421/45838605-fa149700-bcc6-11e8-95cf-8568c08e0d58.png)
